app/src/main/java/org/mozilla/fenix/components/TopSiteStorage.kt

```diff
@@ -88,8 +88,15 @@ class TopSiteStorage(private val context: Context) {
         }
     }
 
+    /**
+     * This is for issue https://github.com/mozilla-mobile/fenix/issues/11660. We prefetch the top
+     * sites for startup so that we're sure that we have all the data available as our fragment is
+     * launched. This method doesn't affect performance since the [getTopSites] runs on the a
```

Comment:
"This method doesn't affect performance" – this seems to contradict the point of this method which is to affect performance (positively).

We probably shouldn't say, "This method doesn't negatively affect performance" because we don't know that for sure (all code that runs negatively affects performance, technically) but we can just mention the facts (this runs on a background thread) and let others draw conclusions from them.